soot-infoflow-summaries/summariesManual/org.xml.sax.XMLReader.xml

```diff
@@ -102,7 +102,15 @@
 			</flows>
 		</method>
 		<method id="void setFeature(java.lang.String,boolean)">
-			<flows>
+			<clears>
+				<clear sourceSinkType="Field"
```

Comment:
Da ist eine Map-artige Struktur. Wenn ein Wert geschrieben wird, kannst du nicht alle anderen Werte ebenfalls wegwerfen. Schau mal in die ``java.util.Map``, wie das dort gemacht wird. Vermutlich fehlt hier einfach ein Constraint.